BookReaderDemo/demo-internetarchive.html

```diff
@@ -2,70 +2,74 @@
-    <script src="../BookReader/jquery.ui.touch-punch.min.js"></script>
-    <script src="../BookReader/jquery.browser.min.js"></script>
-    <script src="../BookReader/dragscrollable-br.js"></script>
-    <script src="../BookReader/jquery.colorbox-min.js"></script>
-    <script src="../BookReader/soundmanager/script/soundmanager2-jsmin.js"></script>
-
-
-    <!-- mmenu for mobile nav -->
-    <script src="../BookReader/mmenu/dist/js/jquery.mmenu.min.js"></script>
-    <script src="../BookReader/mmenu/dist/addons/navbars/jquery.mmenu.navbars.min.js" ></script>
-    <link rel="stylesheet" href="../BookReader/mmenu/dist/css/jquery.mmenu.css" />
-    <link rel="stylesheet" href="../BookReader/mmenu/dist/addons/navbars/jquery.mmenu.navbars.css" />
-
-    <!-- Bookreader -->
-    <script src="../BookReader/BookReader.js"></script>
-    <link rel="stylesheet" href="../BookReader/BookReader.css"/>
-
-    <!-- plugins needed for archive.org, in same order as archive.org -->
-    <script src="../BookReader/plugins/plugin.mobile_nav.js"></script>
-    <script src="../BookReader/plugins/plugin.search.js"></script>
-    <script src="../BookReader/plugins/plugin.chapters.js"></script>
-    <script src="../BookReader/plugins/plugin.tts.js"></script>
-    <script src="../BookReader/plugins/plugin.url.js"></script>
-    <script src="../BookReader/plugins/plugin.autoplay.js"></script>
-    <script src="../BookReader/plugins/plugin.resume.js"></script>
-    <script src="../BookReader/plugins/plugin.archive_analytics.js"></script>
-    <script src="../BookReader/plugins/plugin.text_selection.js"></script>
-
-    
-    <link rel="stylesheet" href="BookReaderDemo.css"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+  <meta name="apple-mobile-web-app-capable" content="yes">
+
+  <!-- JS dependencies -->
+  <script src="../BookReader/jquery-1.10.1.js"></script>
+
+
+  <!-- Bookreader -->
+  <script src="../BookReader/BookReader.js"></script>
+
+  <link rel="stylesheet" href="../BookReader/BookReader.css" />
+
+  <!-- plugins needed for archive.org, in same order as archive.org -->
+  <script src="../BookReader/plugins/plugin.mobile_nav.js"></script>
+  <script src="../BookReader/plugins/plugin.search.js"></script>
+  <script src="../BookReader/plugins/plugin.chapters.js"></script>
+  <script src="../BookReader/plugins/plugin.tts.js"></script>
+  <script src="../BookReader/plugins/plugin.url.js"></script>
+  <script src="../BookReader/plugins/plugin.autoplay.js"></script>
+  <script src="../BookReader/plugins/plugin.resume.js"></script>
+  <script src="../BookReader/plugins/plugin.archive_analytics.js"></script>
+  <script src="../BookReader/plugins/plugin.text_selection.js"></script>
+
+  <!-- CSS only -->
+  <link rel="stylesheet" href="BookReaderDemo.css" />
+
+  <link rel="stylesheet" href="../BookReaderTemplate/styles/BookReader-ia.css" />
+
 </head>
+
 <body>
+  <div id="theatre-ia">
```

Comment:
Can we have these elements created via JS? That way the interface to set up BookReader is the same (you just need a `#BookReader` element)